extensions/amp-analytics/0.1/instrumentation.js

```diff
@@ -514,6 +520,16 @@ export class InstrumentationService {
     this.win_.setTimeout(this.win_.clearInterval.bind(this.win_, intervalId),
         maxTimerLength * 1000);
   }
+
+  isTriggerAllowed_(triggerType, element) {
+    if (!isInIframe(element) ||
```

Comment:
I think at this time we should assume they are 100% one and the same. It's highly likely the rules will be identical. Marking iframe window is fine as well. But in this case, the check is very straightfoward as well so I'd prefer to do this instead of marking. The long answer here - I'd like to eventually move everything to the strategy we are adopting for url replacements with "child service override". But let's learn it in one example first before we try to push it deeper into stack.